controllers/suite_test.go

```diff
@@ -61,9 +60,7 @@ var (
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	RunSpecsWithDefaultAndCustomReporters(t,
-		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
+	RunSpecs(t, "Controller Suite")
```

Comment:
Refactored the same way as them [here](https://github.com/kubernetes-sigs/controller-runtime/pull/1977/files).